dotnet/src/Connectors/Connectors.Memory.Redis/Connectors.Memory.Redis.csproj

```diff
@@ -24,10 +24,6 @@
     <PackageReference Include="NRedisStack" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' Or '$(TargetFramework)' == 'net462' ">
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
```

Comment:
it's not needed anymore, as it depends on System.Text.Json which has a dependency to Microsoft.Bcl.AsyncInterfaces